src/components/MySearch.vue

```diff
@@ -126,18 +127,24 @@ export default {
 
   computed: {
     locationButton() {
-      if (this.value.latitude !== -1 && this.value.longitude !== -1) {
+      if (typeof this.value.latitude !== 'undefined' && typeof this.value.longitude !== 'undefined') {

```

Comment:
why not !== undefined?

Reply:
ohne das typeof

Reply:
```suggestion
      if (this.value.latitude !== undefined && this.value.longitude !== undefined) {

```

Reply:
Ein Stackoverflow Eintrag von 2010 meinte das es so besser ist, neuere Einträge sagen das es mit === undefinded genauso gut ist...

Reply:
Okay. Würde vorschlagen, dass das angepasst werden sollte, da es den Code entschlackt.

Reply:
@dermodmaster done